package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "listecountries",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@react-icons/all-files": "^4.1.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.4.0",
+    "react-redux": "^8.0.2",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.1",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
- [ ] Kindly add the `react-test-renderer` dependency in your packet.json, some of your tests are failing because of this. do `npm install react-test-renderer`

![image](https://user-images.githubusercontent.com/65068771/189245113-18c6041a-f820-4bf3-b2b4-4de7273803e3.png)

Reply:
Thanks